app/templates/discover.hbs

```diff
@@ -26,7 +26,7 @@
         </div>
     </div>
     {{#if queryBody}}
-        <div class="row" style="text-align: center; font-size:75%; margin-top:15px">
+        <div class="row hidden-xs" style="text-align: center; font-size:75%; margin-top:15px">
```

Comment:
It looks like the filter sidebar is completely hidden on small screens. Is it possible to add filters on mobile?